stablehlo/tests/interpret_clamp.mlir

```diff
@@ -0,0 +1,29 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: clamp_op_test_si64
+func.func @clamp_op_test_si64() -> tensor<3xi64> {
+  %min = stablehlo.constant dense<[1, 5, -5]> : tensor<3xi64>
+  %operand = stablehlo.constant dense<[2, 3, -1]> : tensor<3xi64>
+  %max = stablehlo.constant dense<[3, 7, -3]> : tensor<3xi64>
+  %result = stablehlo.clamp %min, %operand, %max : (tensor<3xi64>, tensor<3xi64>, tensor<3xi64>) -> tensor<3xi64>
+  func.return %result : tensor<3xi64>
+  // CHECK-NEXT: tensor<3xi64>
+  // CHECK-NEXT: 2 : i64
+  // CHECK-NEXT: 5 : i64
+  // CHECK-NEXT: -3 : i64
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: clamp_op_test_si64_scalar
+func.func @clamp_op_test_si64_scalar() -> tensor<3xi64> {
+  %min = stablehlo.constant dense<0> : tensor<i64>
+  %operand = stablehlo.constant dense<[2, 3, -1]> : tensor<3xi64>
+  %max = stablehlo.constant dense<1> : tensor<i64>
+  %result = stablehlo.clamp %min, %operand, %max : (tensor<i64>, tensor<3xi64>, tensor<i64>) -> tensor<3xi64>
+  func.return %result : tensor<3xi64>
+  // CHECK-NEXT: tensor<3xi64>
+  // CHECK-NEXT: 1 : i64
+  // CHECK-NEXT: 1 : i64
+  // CHECK-NEXT: 0 : i64
+}
```

Comment:
Let's have two separate tests here - one for scalar min and one for scalar max. I think that's be useful because that would've caught the typo in Ops.cpp called out in the previous comments.

Reply:
Yes, split to:
clamp_op_test_si64_min_scalar
clamp_op_test_si64_max_scalar
clamp_op_test_si64_min_max_both_scalar